docs/user/integration/dynatrace/README.md

```diff
@@ -259,3 +262,33 @@ For easier access from the Kyma dashboard, add links to new navigation under **D
     ```
 
 2. If your Secret has a different name or namespace, then download the file first and adjust the namespace and name accordingly in the 'dataSources' section of the file.
+
+## Use Dynatrace Dashboards
+
+You can view logs, traces, and metrics in Dynatrace dashboards:
+
+- To view the status of the Dynatrace integration with the Kyma Telemetry module, import the file [Telemetry Module Status](./telemetry-resource-metrics.json) as a Dynatrace dashboard. For details on how to import a dashboard, see [Importing Dashboards](https://docs.dynatrace.com/docs/analyze-explore-automate/dashboards-classic/dashboards/dashboard-json#import-dashboard).
+> [!WARNING]
+> Kyma Telemetry resources contain the following custom OpenTelemetry resource attributes:
+> - `k8s.resource.name`
+> - `k8s.resource.group`
+> - `k8s.resource.kind`
+> - `k8s.resource.version`
+> 
+> For the dashboard to function properly, you must [add these attributes to the allow list of OpenTelemetry metrics resource attributes](https://docs.dynatrace.com/docs/ingest-from/opentelemetry/getting-started/metrics/configuration#allow-list).
+
+## Use Dynatrace Alerts
+
+You can create Dynatrace Alerts based on certain metric events to send alerts to your system of choice. 
```

Comment:
That section is generic about how to setup alerting. It should be all about how to setup an alert for the telemetry status, can you add the alert expression please